server/api/users.js

```diff
@@ -42,9 +42,9 @@ router.get('/:id/latestguesses', async (req, res, next) => {
 			where: { userId: req.params.id },
 			include: AcceptedGuess
 		})
-		latestWordle = latestWordle[latestWordle.length - 1]
-		if (latestWordle.acceptedGuesses.length === 5) {
-			latestGuesses === null
+		latestWordle ? (latestWordle = latestWordle[latestWordle.length - 1]) : null
```

Comment:
With this logic, we aren't actually returning the most recent wordle game, just the last wordle game in the database. Which I think logically could work out, but after I seeded the database I realized that the database seeds backwards, so what was being returned was actually the oldest game. This might not matter in the long run but to be safe, I think we should update latestWordle to be 
`let latestWordle = await WordleGame.findOne({
			where: { userId: req.params.id },
			include: AcceptedGuess,
			order: [ [ 'createdAt' ]],
		})`
When I put this into the code it returned the wordle game with the most recent createdAt date + time!